sklearn/compose/_column_transformer.py

```diff
@@ -167,7 +171,14 @@ def get_params(self, deep=True):
         params : mapping of string to any
             Parameter names mapped to their values.
         """
-        return self._get_params('_transformers', deep=deep)
```

Comment:
You shouldn't need to change this. But it seems you've stumbled across a limitation in `_BaseComposition.get_params()`. It should be using `super(_BaseComposition, self).get_params(deep=deep)` instead of `super(_BaseComposition, self).get_params(deep=deep)` to handle the case that there are nested parameters not merely from the collection of estimators.

With that fixed, `set_params` works on `remainder` too.

Reply:
You're welcome to submit a fix for that issue here, or in a separate PR.

Reply:
Update this PR with this fix.